src/test/skript/tests/syntaxes/expressions/ExprVectorBetweenLocations.sk

```diff
@@ -5,11 +5,9 @@ test "vector between locations":
 	set {_vec} to vector between {_loc1} and {_loc2}
 	assert {_vec} is vector(10,10,10) with "vector between locations in same world failed (expected %vector(10,10,10)%, got %{_vec}%)"
 
-	set {_world2} to world("world_the_end")
-	assert {_world2} is set with "no end world found"
-	set {_loc3} to location(10,10,10,{_world2})
+	set {_loc3} to location(10,10,10)
 	set {_vec2} to vector between {_loc1} and {_loc3}
-	assert {_vec2} is vector(10,10,10) with "vector between locations in different worlds failed (expected %vector(10,10,10)%, got %{_vec2}%)"
+	assert {_vec2} is vector(10,10,10) with "vector between locations 1 and 3 (expected %vector(10,10,10)%, got %{_vec2}%)"
```

Comment:
this isn't a very useful test anymore, it's just the same as the assert before it. You can just remove it.